test/MyContract_test.js

```diff
@@ -17,6 +17,8 @@ contract('MyContract', accounts => {
   // the type of data, but will not work on a public testnet.
   // For the latest JobIDs, visit our docs here:
   // https://docs.chain.link/docs/testnet-oracles
+  // Alternativly, visti a node listing service and choose your
```

Comment:
Let's just point to market.link instead of our docs. That page is no longer maintained.